eclipse-mosquitto.yaml

```diff
@@ -0,0 +1,110 @@
+package:
+  name: eclipse-mosquitto
+  version: 2.0.18
+  epoch: 0
+  description: Eclipse Mosquitto - An open source MQTT broker
+  copyright:
+    - license: EPL-2.0 AND BSD-3-Clause
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - cjson
+      - libwebsockets-dev
+      - mosquitto
+      - mosquitto-clients
+      - mosquitto-dev
```

Comment:
Does this need the dev package at runtime?

Reply:
yep its needed because mosquitto-dev install mosquitto-libs and mosquitto-libs++ [here](https://github.com/wolfi-dev/os/blob/dd4749ff04f69d9eff21186fc98e706738d5c9af/mosquitto.yaml#L56-L59) and these lib packages also needed because they contain [libmosquittopp.so.1](https://apk.dag.dev/https/packages.wolfi.dev/os/aarch64/mosquitto-libs++-2.0.18-r3.apk@sha1:19995ff096d407d0589fe0b300605820146196a9/usr/lib/libmosquittopp.so.1) which is added to container image by eclipse-mosquitto [here](https://github.com/eclipse/mosquitto/blob/master/docker/2.0-openssl/Dockerfile#L81).

Reply:
Hmm better would be to add `mosquitto-libs` and `mosquitto-libs++` then - dev packages shouldn't be needed at runtime since they should consist of header and pkgconfig files mainly. Same with `libwebsockets-dev`.

This diff ought to work and still pass the tests.

```
diff --git a/eclipse-mosquitto.yaml b/eclipse-mosquitto.yaml
index bde5b47c7..afa83a018 100644
--- a/eclipse-mosquitto.yaml
+++ b/eclipse-mosquitto.yaml
@@ -9,10 +9,11 @@ package:
     runtime:
       - ca-certificates-bundle
       - cjson
-      - libwebsockets-dev
+      - libwebsockets
       - mosquitto
       - mosquitto-clients
-      - mosquitto-dev
+      - mosquitto-libs
+      - mosquitto-libs++

 var-transforms:
   - from: ${{package.version}}
@@ -51,8 +52,9 @@ test:
   environment:
     contents:
       packages:
-        - libwebsockets-dev
-        - mosquitto-dev
+        - libwebsockets
+        - mosquitto-libs
+        - mosquitto-libs++
         - mosquitto-clients
         - mosquitto
   pipeline:
```